src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -46,13 +46,7 @@
 import static java.util.stream.StreamSupport.stream;
 import static org.mifos.connector.channel.camel.config.CamelProperties.*;
 import static org.mifos.connector.channel.zeebe.ZeebeMessages.OPERATOR_MANUAL_RECOVERY;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.ACCOUNT;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.IS_AUTHORISATION_REQUIRED;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.IS_RTP_REQUEST;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.PARTY_ID;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.PARTY_ID_TYPE;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.TENANT_ID;
-import static org.mifos.connector.channel.zeebe.ZeebeVariables.TRANSACTION_ID;
+import static org.mifos.connector.channel.zeebe.ZeebeVariables.*;
```

Comment:
Let's not do the * import, keep individual imports

Reply:
okay